src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleViewModel.java

```diff
@@ -0,0 +1,141 @@
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.control.ToggleButton;
+import javafx.stage.Stage;
+import javafx.util.Callback;
+
+public class ErrorConsoleViewModel {
+
+    @FXML
+    private Button closeButton;
+    @FXML
+    private Button copyLogButton;
+    @FXML
+    private Button testOutput;
+    @FXML
+    private ToggleButton developerButton;
+    @FXML
+    private ListView<ObservableMessageWithPriority> allMessage;
+
+    private BooleanProperty developerInformation = new SimpleBooleanProperty();
+
+    @FXML
+    private void initialize() {
+
+        ButtonBar.setButtonData(developerButton, ButtonBar.ButtonData.LEFT);
+
+        ObservableList<ObservableMessageWithPriority> masterData = ObservableMessages.INSTANCE.messagesPropety();
+
+        /*
+        that is a ObservableList, which filter the showed message.
+        at first should it show only the Log Entries
+        HERE IT SHOWS NOW NOT CORRECTLY THE LIST ON LISTVIEW. NEED ONE TIPPS PLEASE !
+         */
+        FilteredList<ObservableMessageWithPriority> filteredList = new FilteredList<>(masterData, t -> !t.isFilteredProperty().get());
+        allMessage.setItems(filteredList);
+
+        developerInformation.bind(developerButton.selectedProperty());
+        developerInformation.addListener((observable, oldValue, newValue) -> {
+            if (newValue) {
+                masterData.forEach(message -> message.setIsFiltered(false));
+
+            } else {
+                masterData.forEach(message -> message.setIsFiltered(message.getPriority() != MessagePriority.LOW));
+            }
+        });
+
+        // handler for listCell appearance (example for exception Cell)
+        allMessage.setCellFactory(new Callback<ListView<ObservableMessageWithPriority>, ListCell<ObservableMessageWithPriority>>() {
+            @Override
+            public ListCell<ObservableMessageWithPriority> call(ListView<ObservableMessageWithPriority> listView) {
+                return new ListCell<ObservableMessageWithPriority>() {
+                    @Override
+                    public void updateItem(ObservableMessageWithPriority om, boolean empty) {
+                        super.updateItem(om, empty);
+                        if (om != null) {
+                            setText(om.getMessage());
+                            getStyleClass().clear();
+                            if (om.getPriority() == MessagePriority.HIGH) {
+                                if (developerInformation.getValue()) {
+                                    getStyleClass().add("exception");
+                                }
+                            } else if (om.getPriority() == MessagePriority.MEDIUM) {
+                                getStyleClass().add("output");
+                            } else {
+                                getStyleClass().add("log");
+                            }
+                        }
```

Comment:
add here 
```
} else {
   setText("");
}
```
to solve your problem. Source: http://stackoverflow.com/questions/28384803/listview-using-custom-cell-factory-doesnt-update-after-items-deleted